src/MethodProxies/MpDeactivator.class.st

```diff
@@ -55,8 +55,8 @@ MpDeactivator >> value [
 			During the meta-jump call the handler to tell there was an unwind."
 	thisProcess shiftLevelUp.
 	trapContext := thisContext.
-	[ trapContext := trapContext findNextUnwindContextUpTo: nil ] doWhileFalse: [
-		trapContext method hasPragmaNamed: #trap ].
+	[ trapContext := trapContext findNextUnwindContextUpTo: nil.
+	  trapContext method hasPragmaNamed: #trap ] whileFalse.
```

Comment:
Hi @Gabriel-Darbord why is this an optimization? are you thinking about the double block activation?

Or you mean that it's easier to read?

Anyways, I just ask because curious.
I like your code better :)

Reply:
There is the double block activation and also one message send less (because `doWhileFalse:` just calls `whileFalse`)